director/jobs/templates/jobs_details.html

```diff
@@ -0,0 +1,54 @@
+{% extends 'base.html' %}
+{% block title %}Jobs{% endblock %}
+
+{% block content %}
+    <nav class="level">
+        <div class="level-left">
+            <nav class="breadcrumb" aria-label="breadcrumbs">
+                <ul>
+                    <li><a href="/">Home</a></li>
+                    <li><a href="{% url 'jobs_list' %}">Jobs</a></li>
+                    <li><a href="{% url 'jobs_details' job.id %}">{{job.id}}</a></li>
+                </ul>
+            </nav>
+        </div>
+    </nav>
+
+    <hr>
+
+    <div class="columns is-centered">
+        <div class="column" style="flex: none; width: 228px">
+            <nav class="panel">
+                <a href="#" class="panel-block">Files</a>
+                <a href="#" class="panel-block">Snapshots</a>
+                <a href="{% url 'jobs_list' %}" class="panel-block is-active">Jobs</a>
+                <a href="#" class="panel-block">Sharing</a>
+                <a href="#" class="panel-block">Settings</a>
+            </nav>
+        </div>
+
+        <div class="column">
+          <div class="job-container job-details-container is-flex flex-direction-column border-top-grey-lighter">
```

Comment:
As a user this details view of a job feels a little disappointing (I realise it's just a first step) - I can do less than I can do in the list view i.e. Cancel the job and log messages are repeated in the top and bottom part of the view.

![image](https://user-images.githubusercontent.com/1152336/82177916-ea7fa280-992e-11ea-9dbe-b2e1c41f9a06.png)

Reply:
I've limited the text in the to part of the view which hopefully makes this look like there's less repetition. I've also added the cancel button back in (if the job is incomplete).

I think the message we display in the job summary (as seen on the list & details page), really needs to be more descriptive - as @alex-ketch has described in the wireframes. e.g. more about the position in the queue, type of job *"30274-eLife.rmd to Replication-Study.html"*. I think we need to keep the log out of this summary if possible.

Reply:
Feels like this is a user story to flesh out as an issue of its own?